eva-accession-import/src/test/java/uk/ac/ebi/eva/accession/dbsnp/processors/RenormalizationProcessorTest.java

```diff
@@ -0,0 +1,168 @@
+        assertNonAmbiguousDoesNotChange(5, "", "CC");   // 2:G>GCC
+    }
+
+    @Test
+    public void nonAmbiguousDeletions() throws Exception {
+        assertNonAmbiguousDoesNotChange(3, "C", "");    // 2:GC>G
+        assertNonAmbiguousDoesNotChange(3, "CGC", "");  // 2:GCGC>G
+        assertNonAmbiguousDoesNotChange(5, "CGC", "");  // 4:GCGC>G
+        assertNonAmbiguousDoesNotChange(5, "CC", "");   // 4:GCC>G
+    }
+
+    @Test
+    public void ambiguousInsertions() throws Exception {
+        assertMatchesExpected(3, "", "G", 2, "", "G");   // 2:G>GG
+        assertMatchesExpected(3, "", "CG", 2, "", "GC"); // 2:G>GCG
+        assertMatchesExpected(5, "", "CG", 4, "", "GC"); // 4:G>GCG
+        assertMatchesExpected(7, "", "C", 6, "", "C");   // 6:C>CC
+        assertMatchesExpected(7, "", "CC", 6, "", "CC"); // 6:C>CCC
+        assertMatchesExpected(7, "", "CCC", 6, "", "CCC");   // 6:C>CCCC
+    }
+
+    @Test
+    public void ambiguousDeletions() throws Exception {
+        assertMatchesExpected(3, "CG", "", 2, "GC", ""); // 2:GCG>G
+        assertMatchesExpected(5, "CG", "", 4, "GC", ""); // 4:GCG>G
+        assertMatchesExpected(6, "C", "", 5, "C", "");   // 5:CC>C
+    }
+
+    @Test
+    public void variantsAreCopiedCompletely() throws Exception {
+        // this test is going to transform one list with two variants. One of the variants has the default flag values
+        // for the variant and the other the non default, to be sure the values are being copied
+
+        DbsnpSubmittedVariantEntity variant = new DbsnpSubmittedVariantEntity(SS_ID, HASH, ASSEMBLY, TAXONOMY, PROJECT,
+                                                                              "22", 3, "", "G", RS_ID,
+                                                                              IS_SUPPORTED_BY_EVIDENCE,
+                                                                              MATCHES_ASSEMBLY, ALLELES_MATCH,
+                                                                              VALIDATED, 2);
+        long anotherSsId = SS_ID + 1;
+        String anotherHash = "other";
+        long anotherRsId = RS_ID + 1;
+        String otherProject = "otherProject";
+        DbsnpSubmittedVariantEntity variant2 = new DbsnpSubmittedVariantEntity(anotherSsId, anotherHash, ASSEMBLY,
+                                                                               TAXONOMY, otherProject, "22", 6, "C", "",
+                                                                               anotherRsId, !IS_SUPPORTED_BY_EVIDENCE,
+                                                                               !MATCHES_ASSEMBLY, !ALLELES_MATCH,
+                                                                               !VALIDATED, 3);
+
+        List<DbsnpSubmittedVariantEntity> normalizedVariants = renormalizer.process(Arrays.asList(variant, variant2));
+
+
+        // the version is compared in the equals method, so we have to compare it explicitly
```

Comment:
And created date too? It should be set as the value from the dbSNP database.

Reply:
Variants are manually initialized instead of read from any source, so the created date is not initialized.